src/Microsoft.Framework.Runtime.Abstractions/RuntimeEnvironmentExtensions.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Text;
+
+namespace Microsoft.Framework.Runtime
+{
+    public static class RuntimeEnvironmentExtensions
```

Comment:
Feels like this should be internal or in the common folder instead of in abstractions.